### PR TITLE
 Certification fullpage layout

### PR DESCRIPTION
**Pull request by nolawi**
Connects #4608

### CODE REVIEW

File: client/app/certification/components/EntryPointRedirect.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  render() {
+    let {
+      match
+    } = this.props;
+
+    return <Redirect to={`/certifications/${match.params.vacols_id}/check_documents`} />;
+  }
+}
+
+export default connect()(EntryPointRedirect);
```
Comment (nolawi) [inline client/app/certification/components/EntryPointRedirect.jsx]: copy and paste form `certification`

File: client/app/certification/CancelCertificationConfirmation.jsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import StatusMessage from '../components/StatusMessage';
+import StatusMessage from '@department-of-veterans-affairs/caseflow-frontend-toolkit/components/StatusMessage';
```
Comment (nolawi) [inline client/app/certification/CancelCertificationConfirmation.jsx]: can remove if its beyond the scope of ticket

Comment (nickheiner-usds) [reply client/app/certification/CancelCertificationConfirmation.jsx]: Nah this is good. Thanks.

**Comment by nolawi** [comment]
@nickheiner-usds Looking at this, looks like adding `extrabanner`  to navbar doesn't load the banner. Created and issue to fix this for certification. #4697

**Pull request merged**